guide/16-geoenrichment-guides/part4_enrich_collections_and_sedf.ipynb

```diff
@@ -0,0 +1,2925 @@
+{
```

Comment:
<p>Would be even better if there is a 1-line explanation to what "CS" mean here. For instance, "EDUC" means analysis variables used to describe educational levels.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>